cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceTaskContext.java

```diff
@@ -35,7 +36,7 @@
  */
 @Beta
 public interface MapReduceTaskContext<KEYOUT, VALUEOUT> extends RuntimeContext, DatasetContext,
-  ServiceDiscoverer, PluginContext {
+  ServiceDiscoverer, PluginContext, LocalizationContext {
```

Comment:
Why the task context need to support localization? And how? Task context is only used in Mapper/Reducer, which cannot localize file anymore.